pkg/opcfg/config.go

```diff
@@ -107,8 +109,10 @@ func (o *OperatorConfig) SetFlagArgs() {
 	flag.IntVar(&o.VerticaAutoscalerConcurrency, "verticaautoscaler-concurrency",
 		DefaultVerticaAutoscalerDBConcurrency,
 		"The amount of concurrency to reconcile VerticaAutoscaler CRs")
-	flag.IntVar(&o.EventTriggerConcurrency, "eventtrigger-concurrency", DefaultVerticaDBConcurrency,
+	flag.IntVar(&o.EventTriggerConcurrency, "eventtrigger-concurrency", DefaultEventTriggerDBConcurrency,
 		"The amount of concurrency to reconcile EventTrigger CRs")
+	flag.IntVar(&o.VerticaRestorePointQueryConcurrency, "verticarestorepoint-concurrency", DefaultVerticaRestorePointQueryConcurrency,
+		"The amount of concurrency to reconcile VerticaRestorePointQuery CRs")
```

Comment:
Please correct name of CR:
```suggestion
		"The amount of concurrency to reconcile VerticaRestorePointsQuery CRs")
```

Reply:
Opps, will fix it!